week5-tdd/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -0,0 +1,65 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.util.Collection;
+import java.util.Arrays;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Set;
+
+public final class FindMeetingQuery {
+    private static Comparator<Event> EVENT_ORDER_BY_START = Comparator.comparingLong((Event e) -> e.getWhen().start());
+
+    public static Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+        List<Event> eventsCopy = new ArrayList<>(events);
```

Comment:
why a copy?

Reply:
I wanted to use Collections.sort(), which seems to want a List instead of a Collection